test/bob/api/middleware_test.clj

```diff
@@ -13,7 +13,7 @@
 ;   You should have received a copy of the GNU General Public License
 ;   along with Bob. If not, see <http://www.gnu.org/licenses/>.
 
-(ns bob.middleware-test
+(ns bob.api.middleware-test
```

Comment:
is this change necessary?

Reply:
since I created another test-file in api-ns I moved middleware there, too. I wanted it to be symmetric like in src-folder.

Reply:
makes sense.

Reply:
src/bob/api/middleware.clj -> test/bob/api/middleware_test.clj
src/bob/api/health_check.clj -> test/bobt/api/health_check_test.clj